src/eventHandlers/checkSuite/index.ts

```diff
@@ -105,16 +105,7 @@ const tryMerge = async (
     logInfo(`Pull request is not in a mergeable state: ${mergeableState}.`);
   } else if (merged) {
     logInfo(`Pull request is already merged.`);
-  } else if (
-    mergeStateStatus !== 'CLEAN' &&
-    /*
-     * cspell:ignore merlinnot
-     *
-     * TODO(merlinnot) [2020-09-01] Start pulling the value once it reaches
-     * GA.
-     */
-    mergeStateStatus !== undefined
```

Comment:
@merlinnot I am not really sure what you meant with this TODO but it seems like this condition is irrelevant now.
According to its type, it cannot be undefined.

Reply:
@viestat It's still not GA: https://docs.github.com/en/graphql/reference/objects#pullrequest. We just broke the action completely. `mergeStateStatus` is always undefined (because we're not pulling the value), so `mergeStateStatus !== 'CLEAN'` is always true, so we'll always get "Pull request cannot be merged cleanly." error. Please revert ASAP.

Reply:
Sure thing, thanks. I also did not know that this would merge things automatically skipping the review process.

Reply:
see: https://github.com/ridedott/merge-me-action/pull/496